src/components/database.rs

```diff
@@ -64,6 +105,7 @@ impl Healthy for DatabaseComponent {
         match self
             .db_connection
             .as_ref()
+            .as_ref()
```

Comment:
why double this line?

Reply:
because now `db_connection` is an `Arc` type, we should get a ref to the type inside and then a ref inside the `Option`.